hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/cli/GenericCli.java

```diff
@@ -105,7 +105,6 @@ public Void call() throws Exception {
     throw new MissingSubcommandException(cmd);
   }
 
-  @Override
   public OzoneConfiguration createOzoneConfiguration() {
```

Comment:
This can be made private and only expose `getOzoneConf` to avoid creating multiple `OzoneConfiguration`. We can do this in a follow-up jira.

Reply:
Created HDDS-11992 for this, please feel free to pick up, if you have time.